backend/api/measure/team.go

```diff
@@ -1201,6 +1218,29 @@ func RemoveInvite(c *gin.Context) {
 		ID: &teamId,
 	}
 
+	// check if user role can remove invite
+	invite, err := team.getInviteById(c.Request.Context(), inviteId.String())
```

Comment:
@anupcowkur just store `c.Request.Context()` at the start of the function and then use it wherever required. no need to access it every time.

```go
ctx := c.Request.Context()
// snip
invite, err := team.getInviteById(ctx, inviteId.String())
// snip
```